draft/0004-user-sign-in-experience.md

```diff
@@ -0,0 +1,453 @@
+
+The user profile can be updated by the user during the interaction session. The Logto platform can use the user profile to provide a personalized user experience and security checks based on the SIE settings.
+
+### 4.5 Multi-Factor Authentication(MFA)
+
+MFA is a security process that requires more than one verification factor to verify a user's identity. It can be used as an additional security check during the user interaction flow. Developers can enable MFA using the Logto's SIE settings. Once MFA is enabled, the user will be prompted to provide additional MFA verification records during the interaction session.
+
+## 5. Interaction session
+
+Interaction session is a session that is initiated when a user starts an interaction flow with the Logto platform. Logto platform uses the interaction session to track the user's progress during the interaction flow. The interaction session will be terminated once the user logs out or the session expires.
+
+### 5.1 Interaction session lifecycle
+
+To better understand the user interaction flow, we need to understand the lifecycle of an interaction session.
+
+| Interaction Session State | Description                                                                                    | Restrictions                                                                                                                                              |
+| ------------------------- | ---------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `initiated`               | The interaction session is initiated when a user sends an authentication request to the Logto. | The user can only have one active interaction session at a time. All experience APIs require a valid interaction session.                                 |
+| `identified`              | The user has provided a valid identifier and at least one related verification record.         | The user can be identified by the Logto system. The interaction session can access the user's profile and MFA settings for further security checks.       |
+| `verified`                | The user has provide all necessary verification records.                                       | The user is fully verified by the Logto system. The user can proceed to the user profile completion and authentication process based on the SIE settings. |
+| `authenticated`           | The interaction session has been submitted. Authorization granted                              | The user can access the platform resources and services. The interaction session will be terminated once the user logs out or the session expires.        |
```

Comment:
can we align the format with [other rfcs](https://github.com/logto-io/rfcs/blob/master/draft/0001-organization.md#611-token-parameters)

Reply:
in general i think we can use bullet points for better readability. i'll update 0001 soon